bughouse_console/src/secret_database.rs

```diff
@@ -144,6 +187,19 @@ where
         )
         .execute(&self.pool)
         .await?;
+        sqlx::query(
+            format!(
+                "CREATE TABLE IF NOT EXISTS sessions (
+                    session_id TEXT NOT NULL PRIMARY KEY,
+                    user_name TEXT,
+                    expires_at TIMESTAMP,
```

Comment:
Maybe store last activity time instead?
I guess it only matters if `expire_in` is updated, but in this case my intuition is that you would rather want to apply the new value to all sessions. WDYT?

Reply:
Done.